_posts/2021-08-26-faas-storage-cassandra-astra.md

```diff
@@ -179,9 +181,6 @@ faas-cli secret create astra-clientsecret \
 faas-cli secret create astra-secure-connect \
 --from-file astra-secure-connect
 
-kubectl create secret generic -n openfaas-fn astra-secure-connect \
```

Comment:
This can be removed, I ran into an issue with openfaas that was patched as part of this blog post. faas-cli secret create will work with the latest few versions now :+1: